src/solve/canonicalization.md

```diff
@@ -0,0 +1,9 @@
+# Canonicalization
+
+While the exact approach to canonicalization for this solver will differ slightly
+wrt to lifetimes, please visit [the relevant chalk chapter][chalk] for now.
+
+As of 10<!-- date-check --> January 2023, canonicalization is not yet fully implemented
```

Comment:
You can place the date inside the comment if it bothers the writing style :)

Reply:
not sure what exactly you mean here '^^ feel free to edit this section if you want :grin: i think it's a bit ugly rn but don't care too much.

Reply:
I would be something like...
```
<!-- date-check: jan 2023 -->
As of 10 January 2023,  canonicalization is not yet fully implemented
```